repos/system_upgrade/common/actors/kpatch/actor.py

```diff
@@ -0,0 +1,26 @@
+from leapp.actors import Actor
+from leapp.libraries.stdlib import api
+from leapp.libraries.common.rpms import has_package
+from leapp.models import CopyFile, TargetUserSpacePreupgradeTasks, InstalledRedHatSignedRPM
+from leapp.tags import FactsPhaseTag, IPUWorkflowTag
```

Comment:
move the actor to the `ChecksPhase` (the `ChecksPhaseTag`) - the actor is not getting any facts about the system, but does actions based on collected facts about the system.